File: haskell/problem1.hs
{- Returns all the in an array that are multiples of 3 or 5 -}
multiplesOf :: [Int] -> [Int]
multiplesOf xs = [x | x <- xs, x `mod` 3 == 0 || x `mod` 5 == 0]

answer = sum (multiplesOf [1..999])

File: python/problem1.py
multiples = []

def multiples_of_three_and_five(n):
    for i in range(n):
        if i % 3 == 0 or i % 5 == 0:
        	multiples.append(i)
    print(sum(multiples))


multiples_of_three_and_five(1000)